src/@next/Modal/ModalStyle.ts

```diff
@@ -31,6 +31,7 @@ export const StyledModalContainer = styled.div`
   max-width: calc(100vw - 32px);
   height: fit-content;
   max-height: calc(100vh - 96px);
+  overflow: hidden;
```

Comment:
fixes overflow StyledModalContent square edges to always use StyledModalContainer border-radius of 8px